.gitignore

```diff
@@ -1,8 +1,135 @@
-*.iml
+
+# Created by https://www.toptal.com/developers/gitignore/api/androidstudio
+# Edit at https://www.toptal.com/developers/gitignore?templates=androidstudio
+
+### AndroidStudio ###
+# Covers files to be ignored for android development using Android Studio.
+
+# Built application files
+*.apk
+*.ap_
+
+# Files for the ART/Dalvik VM
+*.dex
+
+# Java class files
+*.class
+
+# Generated files
+bin/
+gen/
+out/
+
+# Gradle files
 .gradle
-/local.properties
-.DS_Store
-/build
```

Comment:
i.e. this stuff should still be in gitignore. There for a reason as some IDEs (Android Studio) or gradle produces stuff you dont want to commit. At least on my setup.
Please cherry pick your preferences for your workflow rather than copy pasting a list from the internet.

Reply:
All three of these point are in the current gitignore file. 
I found toptal from www.gitignore.io files to be pretty useful. 
Do you have any files which are now ignored but should not be?

Reply:
Whoops, made a mistake here. This gitignore file is fine.